src/autoscaler/cf/app_test.go

```diff
@@ -58,6 +54,30 @@ var _ = Describe("Cf client App", func() {
 		})
 	})
 
+	setupStress := func() (*ConnectionWatcher, *ConnectionWatcher) {
```

Comment:
refactored so that we can do stress test on the other api endpoints namely the post call in scaleApp